core/Cargo.toml

```diff
@@ -57,6 +57,8 @@ chrono = { version = "0.4", features = ["serde"] }
 color-eyre = "*"
 lipsum = "0.8.2"
 
+tower = { version = "0.4", features = ["full"] }
```

Comment:
In order to save time on builds (will be helpful when #7 is in) we should aim to have the least amount of features we can.
Could you reduce the feature list to only what is needed for `ServiceBuilder`?

Reply:
yes, I will check what's necessary and update

Reply:
Set the features list to "default" which only enables 2 out of 31 available features. https://docs.rs/crate/tower/0.4.13/features